### PR TITLE
Update filename to use for project info

### PR DESCRIPTION
**Pull request by davidwengier**
Copying Phil in https://github.com/dotnet/roslyn/pull/70156

### CODE REVIEW

**Comment by dibarbet** [comment]
Does this require a razor update?  Or can this go in standalone?

**Comment by davidwengier** [comment]
No Razor update needed, but needs to go in at the same time as a Roslyn update that includes Phil's PR

**Review by phil-allen-msft — APPROVED** [review]

**Comment by phil-allen-msft** [comment]
Approved, but we should probably wait until the Roslyn side PR comes through so that we write & read the same file.

**Comment by davidwengier** [comment]
Tempted to clean this up and move the logic all to Razor, like it says in the comment

**Comment by davidwengier** [comment]
Actually, even if we consolidate all of the logic into the Razor repo, we'd still need dual insertions to change it because the dependency is backwards here, and Roslyn references Razor. Maybe the real cleanup is to flip that, and have Razor deploy a DLL, and pass it to the language server as an extension.

For now I'll merge the Roslyn PR and update this one with a Roslyn bump.

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,6 +8,10 @@
 - Debug from .csproj and .sln [#5876](https://github.com/dotnet/vscode-csharp/issues/5876)
 
 ## Latest
+* Update Roslyn version to 4.9.0-1.23502.8 (PR: [#6447](https://github.com/dotnet/vscode-csharp/pull/6447))
+  * Update Razor project configuration file name (PR: [#70156](https://github.com/dotnet/roslyn/pull/70156))
+
+## 2.5.?
```
Comment (davidwengier) [inline CHANGELOG.md:14]: @dibarbet not sure what the normal process here is. Will it be 2.5.0? .1?

Comment (dibarbet) [reply CHANGELOG.md:14]: it is most likely going to be 2.5.30 unless we have to re-spin for some reason (you can check the release branch build number).  You can use that, or ? and we'll update it when we actually release.

Comment (davidwengier) [reply CHANGELOG.md:14]: I'll leave it then. More likely to notice it needs to be updated if its a question mark

**Comment by davidwengier** [comment]
Finally got the builds to pass, and whilst Windows Sandbox died on me 3 times, I validated on a DevBox and razor seems to continue working fine. My machine is haunted, but that seems unrelated.

**Review by dibarbet — APPROVED** [review]

**Pull request merged**